### PR TITLE
CSPL-892: system resource storage

### PR DESCRIPTION
**Pull request by kumarajeet**
Few changes related validation of doc changes on the supported platformns
- couple of links fix
- removed some random characters
- made content of images block in images.md to be consitent with readme.md contents.

### CODE REVIEW

File: docs/CustomResources.md [hunk 1]
```diff
@@ -236,7 +236,7 @@ the `IndexerCluster` resource provides the following `Spec` configuration parame
 
 ## Example of Guaranteed and Burstable QoS
 
-As given in the [Kubernetes Quality of Service section](readme#using-kubernetes-quality-of-service-classes), you can set up your kubernetes cluster with different Quality of Services by using your Pods’ CPU and Memory resources values. Following are examples of Guaranteed and Burstable QoS:
+As given in the [Kubernetes Quality of Service section](README.md#using-kubernetes-quality-of-service-classes), you can set up your kubernetes cluster with different Quality of Services by using your Pods’ CPU and Memory resources values. Following are examples of Guaranteed and Burstable QoS:
```
Comment (eddikay) [inline docs/CustomResources.md:239]: You can change the CPU and memory resources, and assign different quality of services settings to your pods using the [Kubernetes Quality of Service classes](README.md#using-kubernetes-quality-of-service-classes). Here are some examples of...

File: docs/README.md [hunk 2]
```diff
@@ -103,7 +103,7 @@ information.
 
 ### What Storage Type To Use?
 
-The Kubernetes infrastructure must have access to storage that meets or exceeds the recommendations provided in the Splunk Enterprise storage type recommendations at [Reference Hardware documentation - what storage type to use for a given role?](https://docs.splunk.com/Documentation/Splunk/latest/Capacity/Referencehardware#What_storage_type_should_I_use_for_a_role.3F) In summary, Indexers with SmartStore need NVMe or SSD storage to provide the necessary IIOPs for a successful Splunk Enterprise environment.ßß
+The Kubernetes infrastructure must have access to storage that meets or exceeds the recommendations provided in the Splunk Enterprise storage type recommendations at [Reference Hardware documentation - what storage type to use for a given role?](https://docs.splunk.com/Documentation/Splunk/latest/Capacity/Referencehardware#What_storage_type_should_I_use_for_a_role.3F) In summary, Indexers with SmartStore need NVMe or SSD storage to provide the necessary IIOPs for a successful Splunk Enterprise environment.
```
Comment (eddikay) [inline docs/README.md:106]: ...necessary IOPs...

**Review by vebken-splunk — COMMENTED** [review]
Aside from Edward's comments, it looks good to me.

**Review by vebken-splunk — APPROVED** [review]

File: docs/CustomResources.md (hunk 1 above)
Comment (kumarajeet) [reply docs/CustomResources.md]: Done with minor change to use word "classes" instead of "settings" as QoS are coined as classes in K8S.

**Review by eddikay — APPROVED** [review]

**Review by smohan-splunk — APPROVED** [review]

**Review by smohan-splunk — APPROVED** [review]

**Pull request merged**